src/main/java/IO/DbReader.java

```diff
@@ -1,53 +1,54 @@
-
 package IO;
 
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
+import data.Database;
 import java.io.IOException;
 import java.io.InputStream;
 
-import data.Database;
-
-/* DbReader class accesses blob information to read its contents. */
+/*
+ * DbReader class accesses blob information to read its contents with an
+ * InputStream. If user makes DbReader object and calls the read() function,
+ * user also must call close on the Inputstream returned. Instead, the user
+ * must use the finish() function to handle closing the inputstream and any
+ * other necessary end operations.
```

Comment:
I don't quite understand this comment. Is the user supposed to call `InputStream.close()`, `DbReader.finish()`, or both?

Reply:
just DbReader. ill make more clear